D/DuckDB/build_tarballs.jl

```diff
@@ -28,7 +24,6 @@ make -j${nproc}
 make install
 
 if [[ "${target}" == *-mingw32 ]]; then
-    mkdir -p ${libdir}
```

Comment:
This doesn't hurt, it's actually useful, as it makes sure `${libdir}` is a directory: https://github.com/JuliaPackaging/Yggdrasil/pull/4170#issuecomment-1007568619.  I would _**not**_ remove it.

Reply:
Based on your comment, I think appending /. is also a fix, right?

Reply:
Yes, _**if**_ `${libdir}` is an existing directory in the first place, which `mkdir` ensures.